package.json

```diff
@@ -140,6 +149,9 @@
       }
     ]
   },
+  "extensionDependencies": [
+    "vadimcn.vscode-lldb"
```

Comment:
This only prompted me to install CodeLLDB. The documentation I found on this was a bit unclear, but it seems like there's no way to silently install it as a dependency, without prompting the user first?

Reply:
That's a little frustrating. At least telling the user is worthwhile